### PR TITLE
Display last-seen info in profile

### PR DESCRIPTION
**Pull request by adbenitez**
cc @link2xt

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java [hunk 1]
```diff
@@ -235,11 +236,12 @@ public void onBindHeaderViewHolder(HeaderViewHolder viewHolder, int position) {
         txt = context.getString(R.string.profile_shared_chats);
         break;
       case ItemData.TYPE_PRIMARY_SETTING:
-        if(itemDataContact!=null && itemDataContact.isVerified()) {
-          txt = context.getString(R.string.verified_contact);
+        long lastSeen = (itemDataContact!=null? itemDataContact.getLastSeen() : 0);
+        if(lastSeen==0) {
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:240]:
```suggestion
        if (lastSeen == 0) {
```

i think, this is the style we are mostly using for new code.

(existing code may use different styles, if pulled from some others sources, in general, we do not change the style to to retain the option of easy diffs on updates, same for tabs :)

File: res/values/strings.xml [hunk 2]
```diff
@@ -85,6 +85,8 @@
     <string name="this_month">This month</string>
     <string name="last_week">Last week</string>
     <string name="last_month">Last month</string>
+    <string name="last_seen_at">Last seen at %1$s</string>
+    <string name="last_seen_never">Last seen: never</string>
```
Comment (r10s) [inline res/values/strings.xml:89]:
```suggestion
    <string name="last_seen_never">Last seen: Never</string>
```

usually, we start option names upper case, so maybe this is also better here.

**Review by r10s — APPROVED** [review]
checked the source, lgtm, some very minor nitpicking only :)

File: src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java [hunk 3]
```diff
@@ -235,11 +236,12 @@ public void onBindHeaderViewHolder(HeaderViewHolder viewHolder, int position) {
         txt = context.getString(R.string.profile_shared_chats);
         break;
       case ItemData.TYPE_PRIMARY_SETTING:
-        if(itemDataContact!=null && itemDataContact.isVerified()) {
-          txt = context.getString(R.string.verified_contact);
```
Comment (Hocuri) [inline src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: So, we don't show whether a contact is verified anymore?

I think it should somehow be visible in the profile that a contact is verified

**Comment by Hocuri** [comment]
Nice :tada:!

File: src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java (hunk 3 above)
Comment (r10s) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: it is shown with the blue verified-checkbox, this is what we are doing on desktop/ios as well, so, probably fine.

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: I couldn't find the blue verified-checkbox anywhere, but maybe that's the actual problem

Comment (r10s) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: it is right of the names on android.

ios and desktop shows them on avatars, which is maybe better. however, as this "verified" stuff seems not to be totally settled yet, i am not suggesting to change things here on android :)

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: yip that is what does this PR now the verified checkbox is added to the name, currently it is not show in the chat list, should I also display it there? it is a bit confusing with verified groups but IMHO "verified groups" should be rebranded to "protected group" and use a shield icon for protected chats and let the verified icon for verified contacts then display verified icon everywhere for verified contacts

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: All of this is out of scope for this PR, so consider this approved :)

But at least on my device there is no checkmark shown next to the name
image_2021-11-23_17-10-17

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java:239]: hocuri, check this PR's commits, the checkmark will be shown with this PR it was not show in the DC version you are using that is what this PR changes, it removes the "verified contact" header and instead shows a checkmark

**Pull request merged**